ui/components/app/permission-cell/permission-cell.js

```diff
@@ -107,7 +118,12 @@ const PermissionCell = ({
             'permission-cell__title-revoked': revoked,
           })}
         >
-          {title}
+          {invoker ? (
+            <Box display={Display.InlineBlock} size={Size.MD} marginRight={1}>
+              {extractDomainOrOriginal(invoker)}
+            </Box>
+          ) : null}
+          {invoker ? t('permissionCan', [title.toLowerCase()]) : title}
```

Comment:
Formats the text so that it begins with "can"; e.g.: "X can see the account balance..."